flake.nix

```
@@ -85,12 +89,27 @@
         build = self.hydraJobs.build.${system};
       });
 
+      devShell = forAllSystems (system: self.devShells.${system}.glibc);
+
+      devShells = forAllSystems (system:
+        {
+          glibc = pkgs.mkShell {
+            buildInputs = [ self.packages.${system}.patchelf.inputDerivation];
+          };
+          musl = pkgs.pkgsMusl.mkShell {
+            buildInputs = [ self.packages.${system}.patchelf-musl.inputDerivation];
+          };
```

After switching to:

```suggestion
          glibc = self.packages.${system}.patchelf;
          musl = self.packages.${system}.patchelf-musl;
```

musl just works.